lib/mix/test/mix/tasks/compile.app_test.exs

```diff
@@ -191,34 +208,34 @@ defmodule Mix.Tasks.Compile.AppTest do
       assert_raise Mix.Error, message, fn ->
         Mix.Tasks.Compile.App.run([])
       end
-    end
+    end)
   end
 
   test ".app contains description and registered (as required by systools)" do
     Mix.Project.push(MixTest.Case.Sample)
 
-    in_fixture "no_mixfile", fn ->
+    in_fixture("no_mixfile", fn ->
       Mix.Tasks.Compile.Elixir.run([])
       assert Mix.Tasks.Compile.App.run([]) == :ok
 
-      {:ok, [{_app, _, properties}]} = :file.consult("_build/dev/lib/sample/ebin/sample.app")
+      properties = parse_resource_file(:sample)
       assert properties[:registered] == []
       assert properties[:description] == 'sample'
       assert properties[:applications] == [:kernel, :stdlib, :elixir]
 
       assert Mix.Tasks.Compile.App.run([]) == :noop
-    end
+    end)
   end
 
   test "raise on invalid version" do
     Mix.Project.push(InvalidVsnProject)
 
-    in_fixture "no_mixfile", fn ->
+    in_fixture("no_mixfile", fn ->
       message = "Expected :version to be a SemVer version, got: \"0.3\""
 
       assert_raise Mix.Error, message, fn ->
         Mix.Tasks.Compile.App.run([])
       end
-    end
+    end)
```

Comment:
Why were parens added to all `in_fixture/2` calls?

Reply:
Oh, probably because @fxn ran the formatter from inside lib/elixir and he needs to run from root.

@whatyouhide but honestly, I would just add parens to all in_fixture and all in_tmp calls. WDYT?

Reply:
@josevalim I agree, and remove them from `.formatter.exs`. Seems we avoided doing that initially to limit the already huge amount of changes. But I wouldn't do this here, I can do it as a separate PR/commit.

Reply:
Agreed. @fxn, feel free to leave the parens in the `in_fixture` calls. We will bring all of them back anyway in a later commit. Thank you!